todoapp/src/components/Header.vue

```diff
@@ -1,16 +1,63 @@
 <!-- eslint-disable vue/multi-word-component-names -->
 <script setup lang="ts">
-import DeleteTasks from './DeleteTasks.vue'
+import plus from '../assets/plus.svg'
+import type { TextIdString } from '../types/text'
+import { computed } from 'vue'
+const props = defineProps<{
+  todos: TextIdString[]
+  modelValue: string
+}>()
 
-const emit = defineEmits(['deleteTasks'])
+const emit = defineEmits<{
+  addTodo: any
```

Comment:
Please don't use any type you can type this emit like:
`(e: 'addToto', todo: string): void`